### PR TITLE
Processor & related updates

### PR DESCRIPTION
**Pull request by kkaris**
In this PR:
- The HPOA processors are updated to handle new format of the source data
- Node de-duplication and special character escaping is done for Nodes in the NIH reporter
- Data types are added to the CCLE, ChEMBL and NIH reporter processor output
- Labels are added to the discovery.indra.bio landing page for the two new edge labels (Cell Markers and InterPro Domains)
- Example input in the API docs is added for `get_stmts_for_pubmeds`

The data processed with this code is already in the database, but the service running at discovery.indra.bio needs to be restarted with the new code.

### CODE REVIEW

File: src/indra_cogex/apps/constants.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
     "has_publication": "Project Publications",
     "has_clinical_trial": "Project Trials",
     "has_patent": "Project Patents",
+    "has_marker": "Cell Markers",
+    "has_domain": "InterPro Domains",
```
Comment (cthoyt) [inline src/indra_cogex/apps/constants.py:34]:
```suggestion
    "has_domain": "Protein Domains",
```

File: src/indra_cogex/sources/chembl/__init__.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def get_relations(self) -> Iterable[Relation]:
                 "has_indication",
                 {
                     "source": self.name,
-                    "max_phase:int": max_phase,
+                    "max_phase:float": max_phase,
```
Comment (cthoyt) [inline src/indra_cogex/sources/chembl/__init__.py:94]: why float? to handle nans? typically this is 0-4

File: src/indra_cogex/sources/nih_reporter/__init__.py [hunk 3]
```diff
@@ -241,6 +259,17 @@ def download_files(
             )
 
 
+def clean_text(text: Any) -> Any:
```
Comment (cthoyt) [inline src/indra_cogex/sources/nih_reporter/__init__.py:262]: this function must be rewritten several places throughout the project if you want to try and wrangle them

File: src/indra_cogex/sources/chembl/__init__.py (hunk 2 above)
Comment (kkaris) [reply src/indra_cogex/sources/chembl/__init__.py:94]: Because some phases are =0.5

Comment (kkaris) [reply src/indra_cogex/sources/chembl/__init__.py:94]: See https://chembl.gitbook.io/chembl-interface-documentation/frequently-asked-questions/drug-and-compound-questions#what-is-max-phase

File: src/indra_cogex/sources/nih_reporter/__init__.py (hunk 3 above)
Comment (kkaris) [reply src/indra_cogex/sources/nih_reporter/__init__.py:262]: This is the only place so far (to my knowledge) where this is needed, so maybe we can keep it as is for now?

Comment (cthoyt) [reply src/indra_cogex/sources/nih_reporter/__init__.py:262]: Yes fine for now :)

**Review by bgyori — APPROVED** [review]
Looks great, thanks!

**Pull request merged**